examples/ladder-defaultparams.yaml

```diff
@@ -51,7 +51,7 @@ spec:
         app: autoscaler
     spec:
       containers:
-        - image: gcr.io/google_containers/cluster-proportional-autoscaler-amd64:{LATEST_RELEASE}
```

Comment:
are we even publishing these images? google_containers is _two_ registries and multiple years ago